agent/client/client.go

```diff
@@ -51,8 +52,13 @@ const (
 	backoffMaxDelay      = 15 * time.Second
 	clockDriftWarning    = 5 * time.Second
 	defaultActionTimeout = 10 * time.Second // default timeout for compatibility with an older server
+
+	cacheSize                   = 30_000
+	defaultWindowConnectionTime = "24h"
 )
 
+var cs *connectionSet
```

Comment:
let's create it in `main` and pass it as an argument into `New`

Reply:
Updated